packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -322,6 +322,31 @@ export const getPropValue = (
   return value;
 };
 
+/**
+ * Transforms CSS output into `sheets` and `classNames` ASTs.
+ *
+ * @param cssOutput CSSOutput
+ */
+const transformItemCss = (cssOutput: CSSOutput) => {
+  const sheets: string[] = [];
+  const classNames: t.Expression[] = [];
+
+  cssOutput.css.forEach((item) => {
+    const css = transformCss(getItemCss(item));
+    const className = css.classNames.join(' ');
+
+    sheets.push(...css.sheets);
+
+    if (item.type === 'logical') {
+      classNames.push(t.logicalExpression('&&', item.expression, t.stringLiteral(className)));
+    } else if (item.type === 'unconditional') {
```

Comment:
nit: `unconditional` is fallback case. We can remove if check or probably have switch with last case as `unconditional` without break and logic falling to default case. Actually when I read this, I thought we are skipping other item types. Then I have to jump to the the file where item's type is defined 😅